charters/DataPortal/charter.md

```diff
@@ -0,0 +1,81 @@
+# Data Portal
+
+
+## Description
+The Data Portal provides the overall web-site for the HCA DCP with entry point from https://www.humancellatlas.org/. It provides the user experience for scientific and general users to upload, explore, search and run analyses on human genomic data. The Data Browser ("Explore") application provides a faceted search capability to find and download subsets of this data.
+
+## Objective
+Implementation of the Data Coordination Platform (DCP) website experience. Provide the access point for scientific and general users to search all HCA genetic data in order to identify a subset of the data for download and/or further analysis, providing file/bundle search on multiple clouds at PB scale, with strong public APIs.
+
+## In-scope
+### General
+* UX, visual and branding design with input from HCA leadership (GG, scientific steering committee, etc.)
+* Participation in the HCA-wide standards for Continuous Integration (CI), Continuous Delivery (CD), system logging via Google Analytics, etc.
+* Production operation of the overall Data Portal web-site (see exclusions in Out-of-scope section)
+* Implementation performance scaling
+* DevSecOps compliance as required by the HCA Security Team
+* DevSecOps features for FISMA moderate deployments (authZ/N, logging, auditing, etc) by leveraging other projects (if this becomes a future requirement)
```

Comment:
Change to `authentication and authorization`. This will be clearer to new members of the community than `authZ/N`.